tox.ini

```diff
@@ -104,7 +103,7 @@ install_command =
 [testenv:pinned]
 deps =
     {[pinned]deps}
-    lxml==3.5.0
+    lxml==4.3.0
```

Comment:
Seems like `lxml` is the same version in all envs now, perhaps it could be moved to the main pinned requirements list.